### PR TITLE
Add `lskv_cluster.py` for easily managing docker clusters

### PR DESCRIPTION
**Pull request by jeffa5**
This changes the benchmarks to run lskv in containers compared to raw built objects.

It should also scale better to distributed testing since we now have the logic to do some governance things.

### CODE REVIEW

File: keygenerator.sh [hunk 1]
```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
```
Comment (jumaffre) [inline keygenerator.sh:1]: This file, along with `scurl.sh` are installed in your path as part of the `ccf` Python package.

**Review by heidihoward — APPROVED** [review]

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -4,6 +4,9 @@ on:
   pull_request:
     branches: [main]
 
+env:
+  ACR_REGISTRY: ccfmsrc.azurecr.io
```
Comment (jumaffre) [inline .github/workflows/benchmark.yml:8]: Unused?

File: benchmark/lskv_cluster.py [hunk 3]
```diff
@@ -0,0 +1,677 @@
...
+        run(
+            [
+                "docker",
+                "network",
+                "create",
+                "--subnet",
+                f"{self.subnet_prefix}.0/16",
+                "lskv",
+            ]
+        )
```
Comment (jumaffre) [inline benchmark/lskv_cluster.py:292]: There's [a nice Python library for docker](https://docker-py.readthedocs.io/en/stable/) that CCF uses [here](https://github.com/microsoft/CCF/blob/main/tests/infra/remote_shim.py#L62). I don't think it's worth changing your code to use it, just an FYI.

File: .github/workflows/benchmark.yml (hunk 2 above)
Comment (jeffa5) [reply .github/workflows/benchmark.yml:8]: Yep, left over from a previous try

**Pull request merged**